### PR TITLE
Less core update

### PR DESCRIPTION
**Pull request by BloodBrother**
Nouvelle structure d'utilitaires.

### CODE REVIEW

File: workspace/assets/css/site/util-spacing.less [hunk 1]
```diff
@@ -0,0 +1,446 @@
...
+	max-width: @spacing-larger * 10;
+}
+
+
+.max-width-huge {
+	max-width: @spacing-huge * 10;
+}
+
+// create equal viewport margins on small screens
+[class*="max-width"] {
```
Comment (PascalPiche) [inline workspace/assets/css/site/util-spacing.less]: Very slow selector, to avoid

File: workspace/assets/css/site/util-spacing.less [hunk 2]
```diff
@@ -0,0 +1,446 @@
...
+	});
+}
+
+//====== SPACERS ============
+.top-micro {
+	top: @spacing-micro;
+}
+
+//====== PADDINGS ============
+.padding-large {
```
Comment (PascalPiche) [inline workspace/assets/css/site/util-spacing.less]: Split Vertical and horizontal margin for easiest composition

File: workspace/assets/css/site/util-spacing.less (hunk 1 above)
Comment (PascalPiche) [inline workspace/assets/css/site/util-spacing.less]: Very slow selector and may cause unpredictable behavior. Max width can be used for other than viewport

File: workspace/assets/css/site/util-spacing.less [hunk 3]
```diff
@@ -0,0 +1,446 @@
...
+.padding-right-smaller {
+	padding-right: @spacing-smaller;
+}
+
+.padding-right-smallest {
+	padding-right: @spacing-smallest;
+}
+
+
+.padding-top-smallest {
```
Comment (PascalPiche) [inline workspace/assets/css/site/util-spacing.less]: Media query should be extracted to an other class

**Comment by nitriques** [comment]
We will be doing incremental changes instead

**Pull request closed without merging**